src/views/common/ViewToolbar.tsx

```diff
@@ -0,0 +1,137 @@
+import { Checkbox, IconButton } from '@material-ui/core';
```

Comment:
Again, I'd done some reworking of this guy in the `14-toolbar-synthesis` branch of Figurl, see [updated ViewToolbar.tsx](https://github.com/magland/figurl/blob/14-toolbar-synthesis/src/plugins/sortingview/gui/extensions/common/ViewToolbar.tsx) (that among other things adds support for sliders) and [Toolbars.ts](https://github.com/magland/figurl/blob/14-toolbar-synthesis/src/plugins/sortingview/gui/extensions/common/Toolbars.ts) that defines the relevant types.